public/components/CampaignAnalytics/Analytics/CampaignReferral.js

```diff
@@ -4,6 +4,19 @@ import "react-infinity-menu/src/infinity-menu.css";
 
 export default class CampaignReferral extends React.Component {
 
+  explainImpossibleCTR(ctr) {
+    if (ctr > 1) {
+      const explanation = "The behaviour of the back button in the Safari browser is anomalous;"+
+        " following a link from a front, then using the back button" +
+        " to go back to the front, then following the same link again" +
+        " is being counted as a single impression and two clicks rather than two impressions and two clicks." +
+        " This should have a negligible effect except where the number of impressions is very low," +
+        " as in this case.";
+      return <span className="glyphicon glyphicon-warning-sign" title={explanation}></span>
```

Comment:
I think there are already files in campaign central with icons.

Reply:
Good point.  I'll pull out the icon.